README.md

```diff
@@ -71,9 +71,9 @@ In order to build, package, install and test ST2 in an isolated Vagrant VM, run
 vagrant up $TARGET
 ```
 
-Where `$TARGET` is one of `xenial`, `bionic`, `el7`, or `el8`. If you are using `el8`, comment out the 
- `vm_config.vm.provision :docker` line in the Vagrantfile. There is logic in `setup-vagrant.sh` to 
- install docker in `el8`.
+Where `$TARGET` is one of `xenial`, `bionic`, `focal`, `el7`, or `el8`. If you are using `el8`, comment
+out the `vm_config.vm.provision :docker` line in the Vagrantfile. There is logic in `setup-vagrant.sh`
+to install docker in `el8`.
```

Comment:
Need to also add `focal` to the actual `Vagrantfile`

Reply:
Where is the `Vagrantfile` managed?

Reply:
https://github.com/StackStorm/st2-packages/blob/master/Vagrantfile

Reply:
Looks like Vagrantfile is the only change left for this PR to get merged

https://github.com/StackStorm/st2-packages/blob/23f100cbe709678cbdae81055b8c7dd244c2b883/Vagrantfile#L8-L27